internal/exec/stages/disks/partitions.go

```diff
@@ -323,18 +323,6 @@ func (p PartitionList) Swap(i, j int) {
 	p[i], p[j] = p[j], p[i]
 }
 
-// Expects a /dev/xyz path
-func blockDevHeld(blockDevResolved string) (bool, error) {
-	_, blockDevNode := filepath.Split(blockDevResolved)
-
-	holdersDir := fmt.Sprintf("/sys/class/block/%s/holders/", blockDevNode)
-	entries, err := os.ReadDir(holdersDir)
-	if err != nil {
```

Comment:
Hmm, I am a little worried about removing this check on holder's I have to imagine there are edge cases that would need us to check. Lets flip this change so we make a more direct alteration when doing multi path explicitly.